lib/pagseguro/payment_request.rb

```diff
@@ -82,17 +85,21 @@ def shipping=(shipping)
 
     # Calls the PagSeguro web service and register this request for payment.
     def register
-      path = if @receivers.empty?
-               'checkout'
-             else
-               'checkouts'
-             end
+      request = if @receivers.empty?
+                  Request.post('checkout', api_version, params)
+                else
+                  Request.post_xml('checkouts', api_version, credentials, xml_params)
```

Comment:
ahuah, you will never know if the API accepts xml and x-www-form-urlencoded, I did not test this :P 

Even if it accepts: we will need to test everything in checkout, because of: *I don't trust* an API that changes url and the way to post thing (from `x-www-form-urlencoded` to `xml`) just to add one other attribute (in this case a array).